Client/Get.cs

```diff
@@ -2,27 +2,39 @@
 using System.IO;
 using FluentFTP;
 using ShellProgressBar;
+using static Client.Program;
 
 namespace Client
 {
     public static class Get
     {
-        public static int File(ref FtpClient client, Commands.Get files)
+        public static int File(ref FtpClient client, Commands.Get files, FilePath path)
         {
             if (!client.IsAuthenticated)
             {
                 Console.WriteLine("ERROR: No connection to remote server!");
                 return -1;
             }
 
-            if (!string.IsNullOrEmpty(files.Path))
+            if (!string.IsNullOrEmpty(files.Path) && files?.Files?.Count() == 0)
                 Console.WriteLine("\nDownloading File: " + files.Path);
```

Comment:
Can you move this to `MultipleFiles()`? The count check already exists below as well as the print.

Reply:
Or we can just get rid of it entirely